google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -16,18 +16,28 @@
 """Wrappers for Document AI Page type."""
 
 import dataclasses
-from typing import List
+import html
+from typing import List, Optional, Union
+
+from google.cloud.documentai_toolbox.constants import ElementWithLayout
 
 from google.cloud import documentai
 import pandas as pd
 
+ChildrenElements = Union[
+    List["Paragraph"],
+    List["Block"],
+    List["Token"],
+    List["Line"],
```

Comment:
Since we are referencing something that is inside the file you can't use `Block,Paragraph...etc` i had it with TypeVar so you could use `Block,Paragraph...etc` but Yu-han suggested to just use the quotes.